### DISCUSSION TITLE
Is Anybody Running Beta 3 Yet?

### DISCUSSION THREAD

**Discussion by Acewiza** [Security-Onion-Solutions/securityonion#10501] [2.4] [answered]
My confusion never seems to abate.  Can't find the download....

**Comment by dougburks** [comment, marked as answer]
If you're starting from our blog post at https://blog.securityonion.net/2023/06/security-onion-24-beta-3-release-now.html you should see an Installation section that says:
```
We highly recommend starting with an IMPORT installation as shown at:
https://docs.securityonion.net/en/2.4/first-time-users.html
```

When you go to https://docs.securityonion.net/en/2.4/first-time-users.html, you should see a link to the Download section of the docs:
https://docs.securityonion.net/en/2.4/download.html#download

which says:
```
If downloading our Security Onion ISO image, you can find the download link and verification instructions here: https://github.com/Security-Onion-Solutions/securityonion/blob/2.4/main/VERIFY_ISO.md
```

**Reply by Acewiza** [reply]
It was in the 2nd line!  tks...

**Reply by we-r-robots** [reply]
Is 2.4.3 not available yet? I get a Page not found error when following the link from the Download page.  I am only seeing 2.4.2

**Reply by TOoSmOotH** [reply]
Beta 3 is 2.4.2

RC1 will be 2.4.3 once RC1 is released.
RC2 will be 2.4.4 once RC2 is released.
GA will be 2.4.10 once it is GA.

**Reply by jblackley** [reply]
Any idea how long it usually takes to get to GA? I realize there's probably not a release schedule. But, in general, are we looking at 2 months or 10 months?

**Reply by dougburks** [reply]
RC1 is coming soon! Stay tuned! If you want to help us get to GA faster, then please try out Beta 3 and RC1 and give us feedback on what works and what needs to be improved.

**Comment by Acewiza** [comment]
ISO looks good!  Still not seeing nsm on influxdb, elastic authentication or mdadm on install....

Considering the current state of storage generally speaking, perhaps I should start shifting my paradigm away from raid.  My storage needs are relatively small, but I always liked the ability to expand by re-using smaller drives in raid sets.  Some OS's make it easy to plug-n-play a set anywhere without skipping a beat.

**Reply by dougburks** [reply]
> Still not seeing nsm on influxdb, elastic authentication or mdadm on install....

We'll look into the `mdadm` package, but can you provide more information about the other two?

**Reply by Acewiza** [reply]
So know that I'm just installing and running it - no configuration, except the network details and so forth, necessary for it to run, and reporting observations.  I liked to believe the Influx/NSM thing was mdadm-related at first, but it's on a single drive now:
![InfluxNSM](https://github.com/Security-Onion-Solutions/securityonion/assets/76695490/be150cfb-0bb9-4d97-bfe2-f4bea83ceae7)
Authenticating Elastic, i.e. completing the login dialog only ever does as previously described.  Seems like permissions, because it recognizes different credentials, but does not like the correct ones, with no wrong error returned.  Can't get a screenshot of that one.  ;-)

**Reply by dougburks** [reply]
> I liked to believe the Influx/NSM thing was mdadm-related at first, but it's on a single drive now:

I've created an issue for this:
https://github.com/Security-Onion-Solutions/securityonion/issues/10520

> Authenticating Elastic, i.e. completing the login dialog only ever does as previously described. Seems like permissions, because it recognizes different credentials, but does not like the correct ones, with no wrong error returned. Can't get a screenshot of that one. ;-)

I responded to one of your previous discussions about this:
https://github.com/Security-Onion-Solutions/securityonion/discussions/10210

**Comment by Acewiza** [comment]
Just FYI, I'm a pure honeypot with a hard left political leaning, so the worst kind of traffic from all over comes my way.  But that's way too far down in the weeds for what we're doing right now...

**Comment by dougburks** [comment]
> ISO looks good! Still not seeing nsm on influxdb, elastic authentication or mdadm on install....

I did a test using the 2.4.2 Beta 3 ISO image and it looks like `mdadm` is getting installed properly during ISO installation:

![Screenshot 2023-06-06 at 9 59 40 AM](https://github.com/Security-Onion-Solutions/securityonion/assets/1659467/2356ed90-9156-47e3-ab72-e3a20e46bedb)

After installation, the package is there and I can execute the command:
![image](https://github.com/Security-Onion-Solutions/securityonion/assets/1659467/25d2691a-a9ae-4edd-99b7-a31ec8babcba)

Are you able to provide more information about the installation where you didn't see `mdadm`?

**Reply by Acewiza** [reply]
It's not me that's not seeing it - it's the installer. Sure, it's installed at some point. But it needs to be used by the installer to see any raid sets that might be useful for something like nsm.

Chrome still says "no results" for a single conventional SSD.  Just clicking the "configure" button, not doing anything...
![InfluxBDnsm2](https://github.com/Security-Onion-Solutions/securityonion/assets/76695490/2f2b995a-091b-47cb-b331-d425cb1b3729)

**Reply by dougburks** [reply]
> It's not me that's not seeing it - it's the installer. Sure, it's installed at some point. But it needs to be used by the installer to see any raid sets that might be useful for something like nsm.

When you install our Security Onion ISO image, it asks you the minimum number of questions to get a basic OS installed on the disk. Once you reboot and log into that newly installed OS, you can adjust partitioning however you want to before running through our Setup wizard.

Alternatively, you can always perform a manual installation on top of a standard Rocky Linux installation with your custom parititioning.

> Chrome still says "no results" for a single conventional SSD. Just clicking the "configure" button, not doing anything...

There is definitely an issue with InfluxDB not showing anything for NSM Disk Usage. As I mentioned above, I've created an issue for that and we are looking into it:
https://github.com/Security-Onion-Solutions/securityonion/issues/10520

**Reply by Acewiza** [reply]
OK, same answer as before, no biggie.  But I thought that was the main idea behind an iso install for a complex, dedicated server like this - make it easy and slick for the widest possible audience, ya know?  ;-)
